pages/Transfer.qml

```diff
@@ -366,6 +366,9 @@ Rectangle {
                                 spacing: 0
                                 wrapMode: Text.WrapAnywhere
                                 placeholderText: {
+                                    if(isAndroid) {
+                                      return "4.. / 8..";
+                                    }
```

Comment:
Should this be inside the next if? Otherwise it will always display `4.. / 8..` even on testnet / stagenet.